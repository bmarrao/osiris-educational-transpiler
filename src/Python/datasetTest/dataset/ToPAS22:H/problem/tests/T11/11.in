2
10 30
A B C D E F G H I J
A D
A E
A G
A H
A I
A J
B C
B E
B F
B H
B I
B J
C D
C F
C G
C I
C J
D F
D G
D H
D J
E F
E G
E H
E I
F G
F H
G I
H J
I J
10 15
A B C D E F G H I J
A B
A F
A C
B G
B D
C H
C E
D I
D E
E J
G H
G J
F I
F J
H I
