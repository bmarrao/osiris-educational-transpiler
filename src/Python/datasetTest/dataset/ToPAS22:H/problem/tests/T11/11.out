Possivel: 23040
Possivel: 552960
